server/src/controllers/blog-controller.js

```diff
@@ -0,0 +1,41 @@
+import Blog from '../model/blog.js';
+
+export const updateBlog = async(req,res) =>{
+    try
+    {
```

Comment:
use this type of format: try {

ensure `{` is in same line, reducing the number of lines